updated nav menu (#9)
ksyed-data/renovation-tracker-frontend-2025: documentation/nav-menu.md

> # Dashboard 

Reviewer: You should add a title header and move all these to H2 like this:
```markdown
# Title
## Dashboard
## Room Detection
...
```

- This page will contain a project summary and a brief description of the purpose for the renovation tracker.

> - In the top left corner, there will be 5 different tabs that when clicked will lead the user to that specific page. These pages will include Renovations (potential idea for the future), Properties, Before/After, Room Detection, User Feedback, and the Contact Page. The different pages will be visable and easy to navigate. 

Reviewer: ```suggestion
- In the top left corner, there will be 5 different tabs that when clicked will lead the user to that specific page. These pages will include Renovations (potential idea for the future), Properties, Before/After, Room Detection, User Feedback, and the Contact Page. The different pages will be visible and easy to navigate. 
```

> - There will also be a button in the top right that says "Add Property" and that will allow the user to add the property that they want renovated. 

Reviewer: ```suggestion
- There will also be a button in the top right that says "Add Property" and that will allow the user to add the property that they want to view renovation details on. 
```

# Room detection 
- This page will display all the property images that were uploaded by the user.

> - It will show the AI predicted room types using the images that were provided  by the user. 

Reviewer: ```suggestion
- It will show the AI predicted room types using the images that were provided by the user. 
```

- User will be able to give a thumbs up or thumbs down depending on how the AI performed.
- If a room is classified incorrectly the user will be given the option to correct the AI in order to give the user a better experience.

# Before/After
- This page will display side-by-side before and after images of the renovations.

> - Users will be able to highlight the changes that were made, toggle betwwen rooms and renovation types, and use a slider in order to highlight changes that were made.

Reviewer: ```suggestion
- Users will be able to highlight the changes that were made, toggle between rooms and renovation types, and use a slider in order to highlight changes that were made.
```

- Overlay tools will also be used in order to highlight renovations.

# User Feedback
- The user will also be able to provide an explanation in the User Feedback page as to where the AI went wrong or any other comments regarding their experience with the renovation tool.

# Renovations
- This page will store all previous properties the user has uploaded along with the renovations.

# Contact Page 

> - This page will include headshots of all the collaborators including graduation year and month, major/minor, and potentially a breif description if it's wanted by the collaborator. 

Reviewer: ```suggestion
- This page will include headshots of all the collaborators including graduation year and month, major/minor, and potentially a brief description if it's wanted by the collaborator. 
```
